Odata-docs/webapi-8/tutorials/using-odatautf8jsonwriter-to-improve-serialization-performance.md

```diff
@@ -0,0 +1,40 @@
+---
+title:  "Using ODataUtf8JsonWriter to improve serialization performance"
+description: Learn how to configure ASP.NET Core OData 8 to use Utf8JsonWriter to improve serialization performance. 
+date:   2023-08-11
+ms.date: 08/11/2023
+author: habbes
+ms.author: clhabins
+---
+
+# Using ODataUtf8JsonWriter to improve serialization performance
+
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+`Microsoft.OData.Core` version 7.12.2 introduced a new JSON writer that writes to a `Stream` instead of a `TextWriter`. It also introduced a default implementation of this "stream-based" JSON writer that’s based on .NET’s [`Utf8JsonWriter`](/dotnet/api/system.text.json.utf8jsonwriter). Internally, it's referred to as `ODataUtf8JsonWriter`. This writer is more performant than the default JSON writer used in `Microsoft.OData.Core`. To learn more about the stream-based writer, [visit this page](../../odatalib/v7/using-utf8jsonwriter-for-better-performance.md). In this article, we are going to show you how to configure your ASP.NET Core OData application to use the `ODataUtf8JsonWriter`.
```

Comment:
```suggestion
`Microsoft.OData.Core` version 7.12.2 introduced a new JSON writer that writes directly to a `Stream` rather than through a `TextWriter`. It also introduced a default implementation of this "stream-based" JSON writer that’s based on .NET’s [`Utf8JsonWriter`](/dotnet/api/system.text.json.utf8jsonwriter). Internally, it's referred to as `ODataUtf8JsonWriter`. This writer offers better performance than the default JSON writer used in `Microsoft.OData.Core`. To learn more about the stream-based writer, [visit this page](../../odatalib/v7/using-utf8jsonwriter-for-better-performance.md). In this article, we are going to show you how to configure your ASP.NET Core OData application to use the `ODataUtf8JsonWriter`.
```
Be careful in your use of "stream-based". All writers ultimately write to a stream. The main differentiation is that the `ODataUtf8JsonWriter` writes directly to the stream and rides on the .NET `Utf8JsonWriter`